ansible/tasks/postgres-extensions/19-pg_graphql.yml

```diff
@@ -0,0 +1,33 @@
+# pg_graphql
+- name: pg_graphql - download & install dependencies
+  apt:
+    pkg:
+      - clang-11
+      - build-essential
+      - python
```

Comment:
`python2` is a dependency for installing `libgraphqlparser`. It shouldn't be required by `pg_graphql`. did it cause an issue for you?

Reply:
Ah, this is more of a catch all for all packages that would be used by the extension and any other dependencies.